website/components/Link.js

```diff
@@ -22,6 +23,28 @@ export default function Link({ href, children, extern, extend }) {
         extend,
       }}>
       {children}
+      {extern && (
+        <span title="Opens a new tab">
+          <Box
+            as="svg"
+            aria-hidden="true"
+            focusable="false"
+            xmlns="https://www.w3.org/2000/svg"
+            viewBox="0 0 32 32"
+            extend={{
+              verticalAlign: '-2px',
+              color: 'inherit',
+              width: 16,
+              height: 16,
+              stroke: 'currentcolor',
+              strokeWidth: '2px',
+              display: 'inline-block',
+            }}>
+            <path d="M22 11L10.5 22.5M10.44 11H22v11.56" fill="none"></path>
+          </Box>
+          <VisuallyHidden>(new tab)</VisuallyHidden>
+        </span>
+      )}
```

Comment:
This is an accessibility requirement to mark external links as such (visually and aurally).